edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/version_2019_01_30/ModuleManagementHttpClient.cs

```diff
@@ -186,6 +186,15 @@ public override async Task PrepareUpdateAsync(ModuleSpec moduleSpec)
             }
         }
 
+        public override async Task ReprovisionDeviceAsync()
```

Comment:
Same comment as I added to the managementVersion swagger, it needs a version bump and new generated code for the new version

Reply:
Bumped the version now